tests/e2e/release/cht-config/contact-edit.specs.js

```diff
@@ -58,4 +59,50 @@ describe('Editing contacts with the CHT config', () => {
     await contactPage.selectLHSRowByText(healtchCenterName);
     expect(await contactPage.peopleRows.count()).toBe(0);
   });
+
+  it('should change primary contact', async () => {
+    const contacts = [
```

Comment:
Use the factory objects from rosie instead of defining these within your test. 

Our current list of [objects](https://github.com/medic/cht-core/tree/master/tests/factories/cht) here 

And generating hierarchy in a test
https://github.com/medic/cht-core/blob/6e82151d120e57f2332d6cc1ea78da7a5bae69f9/tests/e2e/cht/pregnancy.spec.js#L21